### ISSUE TITLE
hamming_loss / hamming_score

### ISSUE THREAD

**Issue by agamemnonc** [agamemnonc/sklearn-ext#7]
Remove `labels` argument and include `normalize` argument to make it compatible with the other metrics. 

Update `hamming_score` to behave in a similar fashion to `zero_one_loss`.

See also [this issue](https://github.com/scikit-learn/scikit-learn/issues/11743)

**Comment by agamemnonc** [comment]
Fixed 03d59784cb3c2922476245c1faf7db309e744f47
